app/Http/Controllers/Api/Article/ArticleApiController.php

```diff
@@ -50,6 +51,56 @@ public function store(StoreArticle $request)
             ->transformWith(new ArticleShowTransformer())
             ->toArray();
 
-        return response($article, 201, ['msg' => 'Article store successfully loaded']);
+        return response($article, 201, [ 'msg' => 'Article store successfully loaded' ]);
+    }
+
+    public function show($id)
```

Comment:
Don't forget to add doc block on a function. If you are using VSCode install [PHP DocBlocker ](https://marketplace.visualstudio.com/items?itemName=neilbrayfield.php-docblocker)extension.